package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "leite",
-  "version": "0.2.0",
+  "version": "0.3.0",
```

Comment:
Why did you update pkg version?

Reply:
@mvfsillva

Reply:
Hi @marcelorl, thanks for the explanation, but we generate a release after tests.

Please, return pkg version to `0.2.0`

Reply:
aa I got it...im turning it back to 0.2.0 then...

thank you.

Reply:
@mvfsillva I have just done it. Can you validate it again?

thanks.